src/meta/src/hummock/compaction_scheduler.rs

```diff
@@ -466,13 +476,110 @@ where
 
         true
     }
+
+    async fn on_handle_check_split_multi_group(&self) {
+        let mut group_infos = self
+            .hummock_manager
+            .calculate_compaction_group_statistic()
+            .await;
+        group_infos.sort_by_key(|group| group.group_size);
+        group_infos.reverse();
+        let group_split_limit = self.env.opts.split_check_size_limit;
+        let group_size_limit = group_split_limit * 2;
```

Comment:
how about using const value/config instead of magic number?

Reply:
ok